python/cudf/cudf/core/buffer/buffer.py

```diff
@@ -202,6 +236,20 @@ def __cuda_array_interface__(self) -> Mapping:
             "version": 0,
         }
 
+    @property
+    def _cai(self) -> dict:
```

Comment:
It's verbose, but without the `_readonly` suffix, one may assume that `_cai` and `__cuda_array_interface__` return exactly the same thing?

```suggestion
    def _cuda_array_interface_readonly(self) -> dict:
```

Reply:
Done, 👍